foreshadow/intents/general.py

```diff
@@ -11,6 +13,25 @@
 from ..transformers.smart import SimpleImputer, MultiImputer, Scaler, Encoder
 
 
+def mode_freq(series, count=10):
+    """Computes the mode and the most frequent values
+
+        Args:
+            series (pandas.Series): the series to analyze
+            count (int): the n number of most frequent values
+
+    """
+
+    vc = series.value_counts()
+    if series[~series.isnull()].nunique() == 1:
```

Comment:
Why do we need this special path? The rest of the code should still work even if there is only one unique value, no?